ci/check_style.sh

```diff
@@ -16,124 +16,10 @@ conda activate checks
 
 set +e
```

Comment:
```suggestion
```

now that we've removed the explicit `exit` command at the bottom of this script, this `set +e` should be removed also.